src/Catalyst.Cli.Tests/UnitTests/Commands/Response/AddFileToDfsResponseTests.cs

```diff
@@ -0,0 +1,106 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using Catalyst.Abstractions.Types;
+using Catalyst.Cli.Commands;
+using Catalyst.Cli.Tests.UnitTests.Helpers;
+using Catalyst.Protocol.Rpc.Node;
+using Google.Protobuf;
+using Microsoft.Reactive.Testing;
+using NSubstitute;
+using Serilog;
+using Xunit;
+
+namespace Catalyst.Cli.Tests.UnitTests.Commands.Response
+{
+    public sealed class AddFileToDfsResponseTests
+    {
+        private readonly TestScheduler _testScheduler = new TestScheduler();
+        private readonly ILogger _logger;
+
+        public AddFileToDfsResponseTests() { _logger = Substitute.For<ILogger>(); }
+
+        [Fact]
+        public void AddFileToDfsResponse_Failed_Can_Get_Output()
+        {
+            //Arrange
+            var addFileToDfsResponse = new AddFileToDfsResponse
+            {
+                ResponseCode = ByteString.CopyFrom((byte) FileTransferResponseCodeTypes.Failed.Id)
+            };
+
+            var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
+            var addFileToDfsCommand = new AddFileCommand(null, commandContext, _logger);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'addFileToDfsCommand'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)